tests/unit-tests/cart/cart.php

```diff
@@ -76,12 +76,13 @@ public function test_cart_get_discounted_price() {
 		WC_Tax::_insert_tax_rate( $tax_rate );
 		$product = wc_get_product( $product->id );
 
-		WC()->cart->add_to_cart( $product->id, 1 );
-		WC()->cart->add_discount( $coupon->get_code() );
+		wc_add_to_cart( $product->id, 1 );
+		WC()->cart->add_coupon( $coupon->get_code() );
 
 		WC()->cart->calculate_totals();
-		$cart_item = current( WC()->cart->get_cart() );
-		$this->assertEquals( '24.51', number_format( $cart_item['line_total'], 2, '.', '' ) );
+		$item_totals   = WC()->cart->get_item_totals();
+		$cart_item_key = key( WC()->cart->get_cart() );
+		$this->assertEquals( '24.50', number_format( $item_totals[ $cart_item_key ]->total, 2, '.', '' ) );
```

Comment:
`$cart_item` becomes an object and doesn't contain 'line_total' anymore. Do we need to add a deprecation notice/legacy code so other 3rd party devs don't get something blank back when using it, or are we not worried about it?

In addition -- the total from `$item_totals` comes out to 24.50 (it actually comes out to 24.5045, which rounded becomes 24.50 -- so this actually seems more accurate -- but I wanted to note the value change).